svcs/amigo/table.go

```diff
@@ -21,20 +31,36 @@ type TeamRow struct {
 }
 
 type Scoreboard struct {
-	Chals 	[]string	`json:"challenges"`
+	Chals 	[]ChalRow	`json:"challenges"`
 	TeamRow []TeamRow	`json:"teams"`
 }
 
 func (fd *FrontendData) initTeams(teamId string) []byte {
 
 	teams := fd.ts.GetTeams()
 	rows := make([]TeamRow, len(teams))
+	challenges := make([]ChalRow, 5)
+
+	category := [5]string{"Web exploitation", "Forensics", "Cryptography", "Binary", "Reverse Engineering"}
```

Comment:
It could be nice to have this category string as variable at the beginning of the file. I have added the code above.